website/versioned_docs/version-v1.x/tech-stack.md

```diff
@@ -0,0 +1,44 @@
+---
+title: MACI Tech Stack
```

Comment:
Better name for this page? "Architecture"?

"Tech stack" was the best I could come up with, given it's just an overview of the technologies used vs. explaining the actual architecture (in fact, I could see value in a separate "Architecture" page), but input welcome here!